src/app/shared/chart-and-table/chart-and-table.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container-fluid">
+    <div class="row">
+
+        <div class="col-12">
+            <div class="panel">
+                <highcharts-chart [Highcharts]="Highcharts" [options]="chartOptions"
+                    (chartInstance)="logChartInstance($event)">
+                </highcharts-chart>
+            </div>
+
+            <div class="panel table-wrapper">
+                <ag-grid-angular id="datatable" style="width: 100%; height: 420px;" class="ag-theme-alpine"
```

Comment:
Please replace 420px with more relative value - e.g. 25em. We try to use px only when you need some little spacing.